packages/utils/schema/user-profile.graphql

```diff
@@ -0,0 +1,111 @@
+type ProfileName
+  @createModel(accountRelation: SINGLE, description: "Your name") {
+  name: String! @string(maxLength: 150)
+}
+
+type ProfileAvatar
+  @createModel(
+    accountRelation: SINGLE
+    description: "An image and optional fallback images representing your avatar"
+  ) {
+  original: ImageMetadata!
+  alternatives: [ImageMetadata] @list(maxLength: 3)
+}
+
+type ProfileDescription
+  @createModel(
+    accountRelation: SINGLE
+    description: "A short profile bio / blurb / description"
+  ) {
+  description: String! @string(maxLength: 420)
+}
+
+type ProfileEmoji
+  @createModel(accountRelation: SINGLE, description: "Your spirit emoji") {
+  emoji: String! @string(maxLength: 2)
+}
+
+type ProfileHeaderBackground
+  @createModel(
+    accountRelation: SINGLE
+    description: "An image (3:1 ratio) and optional fallback images to be used as a header background"
+  ) {
+  original: ImageMetadata!
+  alternatives: [ImageMetadata] @list(maxLength: 3)
+}
+
+type ProfileURL
+  @createModel(
+    accountRelation: SINGLE
+    description: "Your home on the internet"
+  ) {
+  url: String! @string(maxLength: 240)
+}
+
+type ProfileHomeLocation
+  @createModel(accountRelation: SINGLE, description: "Where you live") {
+  homeLocation: String! @string(maxLength: 140)
```

Comment:
Does the `!` mean it can't be set to `null`?

I think we should reserve the ability to null-out a user's profile if they request it. Also the ability to request we unpin it.

The right to be forgotten is a really big deal, so if someone wants to burn an identity, they should have that capacity.

Reply:
Yes, that's right. I was thinking that applications would just remove the model entirely, but maybe that's insufficient